CMakeLists.txt

```diff
@@ -394,9 +397,14 @@ else()
 
     # adapt install directory to allow distributing dylibs/frameworks in user's frameworks/application bundle
     # NOTE: it's not required to link against SFML.framework
-    set_target_properties(SFML PROPERTIES
-                          BUILD_WITH_INSTALL_RPATH 1
-                          INSTALL_NAME_DIR "@rpath")
+    set_target_properties(SFML PROPERTIES INSTALL_NAME_DIR "@rpath")
+    if(NOT CMAKE_SKIP_BUILD_RPATH)
+        if (CMAKE_VERSION VERSION_LESS 3.9)
+            set_target_properties(${target} PROPERTIES BUILD_WITH_INSTALL_RPATH TRUE)
+        else()
+            set_target_properties(${target} PROPERTIES BUILD_WITH_INSTALL_NAME_DIR TRUE)
```

Comment:
Did we mean to use `SFML` here instead of `${target}`? I can't find a variable defined with the name `target` in CMakeLists.txt. I just found this out because I started enforcing uninitialized variable warnings.

https://github.com/SFML/SFML/actions/runs/6859641809/job/18652242185#step:14:43